server/src/lib/artefacts.ts

```diff
@@ -0,0 +1,216 @@
+import { existsSync, mkdirSync } from 'fs';
+import { readFile, writeFile, unlink } from 'fs/promises';
+import * as path from 'path';
+import * as zlib from 'zlib';
+
+import { S3 } from '@aws-sdk/client-s3';
+import { config } from 'dotenv';
+
+config();
```

Comment:
Why do you initialize the `dotenv` again? It should be initialized in `index.ts` once right? Is it because ESLint puts the `config()` call after the imports in `index.ts`?

Reply:
Not sure actually, let me remove that